project/README.md.twig

```diff
@@ -15,6 +15,14 @@ Branch | Travis | Coveralls | Scrutinizer |
 {{ stable_branch }}   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     | [![Scrutinizer Status][scrutinizer_stable_badge]][scrutinizer_stable_link] |
 {{ unstable_branch }} | [![Build Status][travis_unstable_badge]][travis_unstable_link] | [![Coverage Status][coveralls_unstable_badge]][coveralls_unstable_link] | [![Scrutinizer Status][scrutinizer_unstable_badge]][scrutinizer_unstable_link] |
 
+{% if is_abandoned %}
+## WARNING: This bundle is abandoned
+
+Since maintainers have lost interest in this project, there is no active support on it.
+
+Feel free to ask if you want to help to keep this project up to date.
+
+{% endif %}
 ## Documentation
 
 Check out the documentation on the [official website](https://sonata-project.org/bundles/{{ website_path }}).
```

Comment:
Not related to this PR, but AFAIK we don't have a valid documentation for all projects

Reply:
Indeed, for example: https://github.com/sonata-project/form-extensions